pkg/quarterdeck/db/models/users.go

```diff
@@ -51,6 +51,7 @@ type User struct {
 const (
 	getUserIDSQL    = "SELECT name, email, password, terms_agreement, privacy_agreement, email_verified, email_verification_expires, email_verification_token, email_verification_secret, last_login, created, modified FROM users WHERE id=:id"
 	getUserEmailSQL = "SELECT id, name, password, terms_agreement, privacy_agreement, email_verified, email_verification_expires, email_verification_token, email_verification_secret, last_login, created, modified FROM users WHERE email=:email"
+	getUserTokenSQL = "SELECT id, name, email, password, terms_agreement, privacy_agreement, email_verified, email_verification_expires, email_verification_secret, last_login, created, modified FROM users WHERE email_verification_token=:token"
```

Comment:
Should we omit the password when selecting the user details for a token? It's not needed for this endpoint and reducing the amount of time even a hashed password in memory is probably a good idea.

Reply:
Good idea

Reply:
But the problem is that the db `Save()` method on the user requires the entire user object.

Reply:
Hmm … ok; I'll create a technical debt story for this.